<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VedioDlg</class>
 <widget class="QDialog" name="VedioDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCommandLinkButton" name="commandLinkButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>187</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>881</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>视频</string>
   </property>
   <widget class="QListWidget" name="listWidgetFiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>211</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QmyVideoWidget" name="videoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>621</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btnPlay">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/images/620.bmp</normaloff>:/images/images/620.bmp</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStop">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/images/624.bmp</normaloff>:/images/images/624.bmp</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPause">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/images/622.bmp</normaloff>:/images/images/622.bmp</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPush">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>0</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>全屏</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/images/237.GIF</normaloff>:/images/images/237.GIF</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSound">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/images/volumn.bmp</normaloff>:/images/images/volumn.bmp</iconset>
     </property>
    </widget>
    <widget class="QSlider" name="sliderVolumn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>160</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPre">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>0</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/images/610.bmp</normaloff>:/images/images/610.bmp</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSuf">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>0</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/images/612.bmp</normaloff>:/images/images/612.bmp</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <widget class="QSlider" name="sliderPosition">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>1</y>
       <width>441</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labRatio">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>0</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>进度</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBoxFile">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>401</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelFile">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>文件路径:</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btnFile">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/images/316.bmp</normaloff>:/images/images/316.bmp</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemove">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/images/318.bmp</normaloff>:/images/images/318.bmp</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmyVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qmyvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
